source/material_model/rheology/visco_plastic.cc

```diff
@@ -148,12 +148,22 @@ namespace aspect
                             + Utilities::to_string(adiabatic_temperature_gradient_for_viscosity) + ") and pressure ("
                             + Utilities::to_string(in.pressure[i]) + ")."));
 
-              // Step 1a: compute viscosity from diffusion creep law, at
-              // least if it is going to be used
+              // Step 1a: compute viscosity from diffusion creep law, at least if it is going to be used
+
+              // Determine whether to use adiabatic pressure instead of full pressure (default) when
+              // calculating creep viscosity. Because in some models, the full pressure may have an
```

Comment:
```suggestion
              // calculating creep viscosity. 
```

Reply:
Also remove the next two lines. This documentation is certainly helpful, but I think it should go into the parameter description rather than here in the code.